llvm/lib/Target/X86/Disassembler/X86Disassembler.cpp

```diff
@@ -1134,6 +1134,21 @@ static int getInstructionIDWithAttrMask(uint16_t *instructionID,
   return 0;
 }
 
+static bool isNFnotMap4(InternalInstruction *insn) {
+  // Promoted BMI instrs below has nf version.
+  if (insn->opcodeType == THREEBYTE_38 &&
+      ppFromXOP3of3(insn->vectorExtensionPrefix[2]) == VEX_PREFIX_NONE) {
```

Comment:
I don't understand. Why do you call function `ppFromXOP3of3` for NF instructions? I remember intel does not support `XOP`.

Reply:
It's my mistake, I copied the wrong function.